meta-openbmc-machines/meta-openpower/conf/machine/include/openpower.inc

```diff
@@ -4,6 +4,7 @@ OBMC_MACHINE_FEATURES += "\
         obmc-phosphor-sensor-mgmt \
         obmc-phosphor-flash-mgmt \
         obmc-phosphor-event-mgmt \
+        obmc-phosphor-user-mgmt \
```

Comment:
Ok nevermind.  You did make this a distro feature.  That's great.  But this line shouldn't be necessary - the package should get pulled in from the distro configuration file that you updated below.  Did the package not get pulled in if you don't do this?